skfda/exploratory/visualization/_magnitude_shape_plot.py

```diff
@@ -131,20 +134,24 @@ class MagnitudeShapePlot:
                         [-1. ],
                         [-1. ],
                         [-1. ]]]),
-                grid_points=(array([ 0.,  2.,  4.,  6.,  8., 10.]),),
+                grid_points=(array([  0.,   2.,   4.,   6.,   8.,  10.]),),
                 domain_range=((0.0, 10.0),),
-                ...),
+                dataset_name=None,
+                argument_names=(None,),
+                coordinate_names=(None,),
+                extrapolation=None,
+                interpolation=SplineInterpolation(interpolation_order=1, smoothness_parameter=0, monotone=False)),
             multivariate_depth=None,
             pointwise_weights=None,
             alpha=0.993,
             points=array([[ 1.66666667,  0.12777778],
-                          [ 0.        ,  0.        ],
-                          [-0.8       ,  0.17666667],
-                          [-1.74444444,  0.94395062]]),
+                [ 0.        ,  0.        ],
```

Comment:
Why did you remove indentation?

Reply:
because it had changed, if i didnt touch it would be different

Reply:
Whitespace should be ignored in the comparisons, if the setup.cfg is correct.

Reply:
ok, just rrevised it and corrected it, the params are correct in setup.cfg too